docker/images/kuma/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mdnwebdocs/kuma_base:latest
+FROM mdnwebdocs/kuma_base:latest@sha256:f72cd16767172d465dfbc4ea62fe28a89dca48cc117b418916e84adb9701e58b
```

Comment:
This shouldn't be pinned, since for example, in the Jenkins build of the master branch we rely on the Docker build for `kuma` picking-up the immediately preceding build of `mdnwebdocs/kuma_base` (see https://github.com/mozilla/kuma/blob/42c8bb2a76e35618fd27c24db297a6e5d1cba88a/Jenkinsfiles/master.groovy#L14).